openspec/changes/fix-share-extension-credentials/design.md

```
@@ -0,0 +1,87 @@
+## Context
+
+The share extension credential pipeline had three layers:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)